README.md

```diff
@@ -120,6 +120,7 @@ jobs:
 | `datadog_site`      | string | _optional_  | The Datadog site. For users in the EU, set to `datadoghq.eu`. For example: `datadoghq.com` or `datadoghq.eu`. **Default:** `datadoghq.com`.                                                              |
 | `config_path`       | string | _optional_  | The global JSON configuration is used when launching tests. See the [example configuration][4] for more details. **Default:** `datadog-ci.json`.                                                         |
 | `variables`         | string | _optional_  | Comma-separated list of global variables to use for Synthetic tests. For example: `START_URL=https://example.org,MY_VARIABLE=My title`. **Default:** `[]`.                                               |
+| `tunnel`            | boolean| _optional_  | Enable tunnel to interact with Datadog API. **Default:** `false`.                                                                                                                                        |
```

Comment:
This does not really describe the tunnel feature, we can reuse the [datadog-ci documentation](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#global-configuration-file-options) as is:
```suggestion
| `tunnel`            | boolean| _optional_  | Use the [secure tunnel][9] to execute your test batch. **Default:** `false`.                                                                                                                             |
```

and add at the end of the README:
```md
[9]: https://docs.datadoghq.com/continuous_testing/testing_tunnel/
```